pages/index.vue

```diff
@@ -1,5 +1,23 @@
 <template>
-  <div>
-    <h1>home</h1>
+  <div
+    v-for="movieOrTvShow in movies"
+    class="group col-span-6 md:col-span-4 lg:col-span-3 xl:col-span-3 transform transition duration-300 ease-in-out hover:bg-gray-800 hover:scale-95 rounded-lg"
+  >
+    <CardComponent
+      :imageEndpoint="movieOrTvShow?.poster_path"
+      :type="movieOrTvShow?.media_type"
+    />
   </div>
 </template>
+<script setup lang="ts">
+import { getTrendingAll } from '~/utils/home.rest';
+
+const config = useRuntimeConfig();
+
+const { $api } = useNuxtApp();
+
+const useGetTrendingAll = getTrendingAll($api);
+const { data } = await useAsyncData(() => useGetTrendingAll.get());
+const movies = data.value?.results;
```

Comment:
This requisition call movies and series results, chose a better name of variable.